<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор картинок v1.0.3</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/rsc/resize.png</normaloff>:/rsc/resize.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="checkedPb">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Отметить все</string>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="rsc.qrc">
                <normaloff>:/rsc/unchecked_checkbox.png</normaloff>:/rsc/unchecked_checkbox.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="showCheckedPb">
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="rsc.qrc">
                <normaloff>:/rsc/eyes.png</normaloff>:/rsc/eyes.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="descImgPb">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Отобразить свойства у всех изображений</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="rsc.qrc">
                <normaloff>:/rsc/description.png</normaloff>:/rsc/description.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="imgListLabel">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Картинок: 0 шт.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar">
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="imgsLw">
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="flow">
             <enum>QListView::LeftToRight</enum>
            </property>
            <property name="resizeMode">
             <enum>QListView::Fixed</enum>
            </property>
            <property name="layoutMode">
             <enum>QListView::SinglePass</enum>
            </property>
            <property name="modelColumn">
             <number>0</number>
            </property>
            <property name="selectionRectVisible">
             <bool>false</bool>
            </property>
            <property name="currentRow">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearPb">
            <property name="text">
             <string>Очистить список</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Параметры преобразования</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Папка с изображениями</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="imgDirLe">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="folderImgPb">
            <property name="text">
             <string>Обзор</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Предустановки:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="resolutionCb"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>15</height>
               </size>
              </property>
              <property name="text">
               <string>Ширина:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="wImgLe">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>15</height>
                 </size>
                </property>
                <property name="text">
                 <string>Высота:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="hImgLe">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="saveResolCb">
            <property name="text">
             <string>Сохранить оригинальное разрешение</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Способ преобразования</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QRadioButton" name="saveRatioRb">
             <property name="text">
              <string>Сохранить соотношение сторон</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="ignorRb">
             <property name="text">
              <string>Игнорировать соотношение сторон</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Режим преобразования</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QRadioButton" name="fastScaleRb">
             <property name="text">
              <string>Быстрый</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="smoothScaleRb">
             <property name="text">
              <string>Качественный</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Папка сохранения</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="saveDirLe">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveDirPb">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>обзор</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="openDirChB">
            <property name="text">
             <string>Открыть папку сохранения по завершению операции</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="resultFolderCb">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Создать папку Result</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="runPb">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Выполнить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Разработчик</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>folderImgPb</tabstop>
  <tabstop>ignorRb</tabstop>
  <tabstop>saveRatioRb</tabstop>
  <tabstop>saveDirPb</tabstop>
  <tabstop>checkedPb</tabstop>
  <tabstop>descImgPb</tabstop>
  <tabstop>imgsLw</tabstop>
  <tabstop>clearPb</tabstop>
  <tabstop>saveDirLe</tabstop>
  <tabstop>openDirChB</tabstop>
  <tabstop>runPb</tabstop>
  <tabstop>imgDirLe</tabstop>
 </tabstops>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
